scripts/Gothic2Notr.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
+
+function cdroot()
+{
+  cd ..; 
+  echo ${PWD}
+}
+
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-export LD_LIBRARY_PATH="$DIR/lib:$LD_LIBRARY_PATH"
+
+
+export LD_LIBRARY_PATH=(cdroot)+"/lib:$LD_LIBRARY_PATH"
 if [[ $DEBUGGER != "" ]]; then
-  exec $DEBUGGER --args "$DIR/bin/Gothic2Notr" "$@"
```

Comment:
CMake build script suppose to put all outputs in /opengothic, including all *.so, that means `export LD_LIBRARY_PATH=$DIR:$LD_LIBRARY_PATH"`, is enough and `cdroot` is not required at all